web/html/src/manager/maintenance/schedule-edit.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable */
+'use strict';
+
+import React, {useState, useEffect, forwardRef, useImperativeHandle} from "react";
+import {Check} from "components/input/Check";
+import {Form} from "components/input/Form";
+import {Text} from "components/input/Text";
+import {Radio} from "components/input/Radio";
+import {Button} from "components/buttons";
+import {Combobox} from "components/combobox";
+
+const MaintenanceScheduleEdit = forwardRef((props, ref) => {
+    const [scheduleName, setScheduleName] = useState("");
+    const [scheduleType, setScheduleType] = useState("SINGLE");
+    const [calendarName, setCalendarName] = useState("");
+    const [calendarAdded, setCalendarAdded] = useState(false);
+    const [strategy, setStrategy] = useState(false);
```

Comment:
Same as the calendar form, `scheduleName`, `scheduleType` and `strategy` can be used directly within the `model` object.